acq4/util/micromanager.py

```diff
@@ -31,7 +62,38 @@ def getMMCorePy(path=None):
                 import MMCorePy
             finally:
                 sys.path.pop()
-
+    
         _mmc = MMCorePy.CMMCore()
 
+
+    if _mmc.getSystemState().size()<=12:  # hack to see if a real configuration is already loaded...
+
+        if mm_config_file is not None:
+            try:
+                _mmc.loadSystemConfiguration(mm_config_file)
+                _mmc.getVersionInfo()
+            except:
+                print(_mmc.getDeviceAdapterSearchPaths())
+                raise ValueError("micromanager loadSystemConfiguration failed!")
+        else:
+            print("no micromanager configuration defined or loaded")
+    else:
+        print("micromanager configuration already loaded")
```

Comment:
Seems like this should only be called once when CMMCore is instantiated, rather than every time getMMCorePy is called.